ext/opcache/config.m4

```diff
@@ -18,6 +18,12 @@ PHP_ARG_ENABLE([opcache-jit],
   [yes],
   [no])
 
+PHP_ARG_WITH([opcache-capstone],,
+  [AS_HELP_STRING([--with-opcache-capstone],
```

Comment:
I think ``--with-capstone`` should be enough.
In general, we need the same flag for Windows.
it's great you kept udis86 yet.

Reply:
I think disambiguation of flags isn't a bad thing but either works for me.
https://github.com/php/php-src/pull/10952

Reply:
I had second thoughts about `--with-capstone` after seeing that all of the FPM flags were named `--with-fpm-foo`, but FWIW I don't mind either way.